database/schema.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose')
+
+mongoose.connect('mongodb://localhost/lizardboard')
+const db= mongoose.connection
+db.on('error', console.error.bind(console,'connection error:'))
```

Comment:
This is connection configuration stuff, probably belongs in a separate module (see comment above).